pageobjects/loginPage.js

```diff
@@ -2,61 +2,80 @@ module.exports = {
   url: function useEnvironmentUrl() {
     return this.api.launch_url;
   },
+
   elements: {
-    loginPage: '.login',
+    landingPage: {
+      selector: ".card-pf",
+    },
+    usernameInput: {
+      selector: "#username", // ID selector for the username input field
+    },
+    passwordInput: {
+      selector: "#password", // ID selector for the password input field
+    },
+    submitBtn: {
+      selector: 'input[name="login"]', // Name selector for the submit btn field
+    },
+    rememberChk: {
+      selector: "#rememberMe", //ID selector for the rememberMe checkbox field
+    },
+    nextBtn: {
```

Comment:
I'm slightly confused by the naming here. Does the button say "next".  your comment also says its for the remember me checkbox.